### ISSUE TITLE
floatBitsToInt functions not supported at #extension GL_ARB_gpu_shader5.

### ISSUE THREAD

**Issue by ZhiqianXia** [KhronosGroup/glslang#2698]
**Shader**
```
#version 150
#extension GL_ARB_gpu_shader5 : require
uniform int expected_value;
uniform float value;
out     vec4 result;
void main()
{
    result = vec4(1.0, 1.0, 1.0, 1.0);
    int ret_val = floatBitsToInt(value);
    if (expected_value != ret_val){  result = vec4(0.0, 0.0, 0.0, 0.0); }

    switch (gl_VertexID)  {
      case 0: gl_Position = vec4(-1.0, 1.0, 0.0, 1.0); break;
      case 1: gl_Position = vec4( 1.0, 1.0, 0.0, 1.0); break;
      case 2: gl_Position = vec4(-1.0,-1.0, 0.0, 1.0); break;
      case 3: gl_Position = vec4( 1.0,-1.0, 0.0, 1.0); break;
    }
}
```

refs:
https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_gpu_shader5.txt

**Comment by ZhiqianXia** [comment]
floatBitsToInt has also been forked off into a separateextension (ARB_shader_bit_encoding).

**Comment by ZhiqianXia** [comment]
Hi, 
    Recenly, I Review the issue and ask a question to [VK-GL-CTS](https://github.com/KhronosGroup/VK-GL-CTS/issues/297),
their respond is : 
Accoding to the [spec](https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_gpu_shader5.txt), the floatBitsToInt
not only support at extension ARB_shader_bit_encoding , but also supoort at extension  GL_ARB_gpu_shader5 . 


    _(21) Should we provide mechanisms allowing shaders to examine or set the
         bit representation of floating-point numbers?

      RESOLVED:  Yes, we will provide functions to convert single-precision
      floats to/from signed and unsigned 32-bit integers.  The
      ARB_gpu_shader_fp64 extension will provide similar functionality for
      double-precision floats.  We chose to adopt the Java naming convention
      here -- converting a single-precision float to/from a signed integer is
      accomplished by the functions floatBitsToInt() and intBitsToFloat().

      Note that this functionality has also been forked off into a separate
      extension (ARB_shader_bit_encoding) that can be exported on
      implementations capable of performing such conversions **but not capable
      of the full feature set of this extension and/or OpenGL 4.0.**_

**Comment by ZhiqianXia** [comment]
Fix the issue at https://github.com/KhronosGroup/glslang/pull/2859 .

**Comment by greg-lunarg** [comment]
Fixed by #2859
